hearth/lib/hearth/event_stream/header_value.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Hearth
+  module EventStream
+    # Represents a HeaderValue in an EventStream::Message.
+    class HeaderValue
```

Comment:
Is there no concept of "field" here? But also why cant these be fields?

Reply:
These are different than our existing `Field` because they need to encode the type.  I guess we could rename these to use the word Field instead of header, but the specs all use "Header" (and eventstreams have no concept of trailer).